README.md

```diff
@@ -31,15 +31,17 @@ Background and documentation is available at <https://iceberg.apache.org>
 
 Iceberg is under active development at the Apache Software Foundation.
 
-The core Java library that tracks table snapshots and metadata is complete, but still evolving. Current work is focused on adding row-level deletes and upserts, and integration work with new engines like Flink and Hive.
+The [Iceberg format specification][iceberg-spec] is stable.
```

Comment:
I think its good, but we will have more versions like v2, v3.  Not sure if its worth mentioning like 'is versioned'?

Reply:
That's a good suggestion @szehon-ho, what do you think of:

```suggestion
The [Iceberg format specification][iceberg-spec] is stable and new features are added with each version.
```

Reply:
Yes, great!  It's what i was thinking but couldnt find the words!

Reply:
Nice, thanks for letting us know! 👍